src/main/groovy/edu/oregonstate/mist/recommendations/db/BatchScoreDAO.groovy

```diff
@@ -37,10 +37,32 @@ public interface BatchScoreDAO extends Closeable {
                           @Bind("minScore") Integer minScore)
 
     @SqlUpdate("""
+                UPDATE BATCH_SCORE
+                  SET YEAR = :year , MIN_SCORE = :minScore,
+                      STUDENT_POOL_ID = (SELECT ID FROM STUDENT_POOL
```

Comment:
What happens with the previous pool? Is it ok for the previous pool to be left unused? Should the previous pool be deleted if there are no references to it?